packages/react/src/patterns/sections/CardSectionSimple/__stories__/CardSectionSimple.stories.js

```diff
@@ -26,6 +26,12 @@ storiesOf('Patterns (Sections)|CardSectionSimple', module)
         heading={text('Heading (required):', 'Aliquam condimentum interdum')}
         theme={select('theme', themes, themes.white)}
         cards={object('Data', cards.CardSectionSimple)}
+        cta={{
```

Comment:
Do we need a way to make this optional? as a knob?

Reply:
> Do we need a way to make this optional? as a knob?

may be we could add a check box in knobs by checking enable cta prop .